spec/collator.html

```diff
@@ -49,46 +49,39 @@
-        2. Set collator.[[initializedIntlObject]] to *true*.
-        3. Let _requestedLocales_ be CanonicalizeLocaleList(_locales_).
-        4. ReturnIfAbrupt(_requestedLocales_).
-        5. If _options_ is *undefined*, then
+        1. Set collator.[[initializedIntlObject]] to *true*.
+        1. Let _requestedLocales_ be ? CanonicalizeLocaleList(_locales_).
+        1. If _options_ is *undefined*, then
           1. Let _options_ be ObjectCreate(*%ObjectPrototype%*).
-        6. Else
-          1. Let _options_ be ToObject(_options_).
-          1. ReturnIfAbrupt(_options_).
-        7. Let _u_ be GetOption(_options_, *"usage"*, *"string"*, «*"sort"*, *"search"*», *"sort"*).
-        8. ReturnIfAbrupt(_u_).
-        9. Set _collator_.[[usage]] to _u_.
-        10. If _u_ is *"sort"*, then
-          1. Let _localeData_ be the value of *%Collator%*.[[sortLocaleData]];
-        11. Else
+        1. Else
+          1. Let _options_ be ? ToObject(_options_).
+        1. Let _u_ be ? GetOption(_options_, *"usage"*, *"string"*, «*"sort"*, *"search"*», *"sort"*).
+        1. Set _collator_.[[usage]] to _u_.
+        1. If _u_ is *"sort"*, then
+          1. Let _localeData_ be the value of *%Collator%*.[[sortLocaleData]].
+        1. Else,
           1. Let _localeData_ be the value of *%Collator%*.[[searchLocaleData]].
-        12. Let _opt_ be a new Record.
-        13. Let _matcher_ be GetOption(_options_, *"localeMatcher"*, *"string"*, «*"lookup"*, *"best fit"*», *"best fit"*).
-        14. ReturnIfAbrupt(_matcher_).
-        15. Set _opt_.[[localeMatcher]] to _matcher_.
-        16. For each row in <emu-xref href="#table-1">Table 1</emu-xref>, except the header row, do:
+        1. Let _opt_ be a new Record.
+        1. Let _matcher_ be ? GetOption(_options_, *"localeMatcher"*, *"string"*, «*"lookup"*, *"best fit"*», *"best fit"*).
+        1. Set _opt_.[[localeMatcher]] to _matcher_.
+        1. For each row in <emu-xref href="#table-1">Table 1</emu-xref>, except the header row, do:
           1. Let _key_ be the name given in the Key column of the row.
           1. Let _prop_ be the name given in the Property column of the row.
           1. Let _type_ be the string given in the Type column of the row.
           1. Let _list_ be a List containing the Strings given in the Values column of the row, or *undefined* if no strings are given.
-          1. Let _value_ be GetOption(_options_, _prop_, _type_, _list_, *undefined*).
-          1. ReturnIfAbrupt(_value_).
+          1. Let _value_ be ? GetOption(_options_, _prop_, _type_, _list_, *undefined*).
           1. If the string given in the Type column of the row is *"boolean"* and value is not *undefined*, then
-            1. Let _value_ be ToString(_value_).
-            1. ReturnIfAbrupt(_value_).
+            1. Let _value_ be ? ToString(_value_).
           1. Set _opt_.[[&lt;_key_&gt;]] to _value_.
```

Comment:
opened an issue to discuss the details: https://github.com/tc39/ecma402/issues/56